internal/dag/builder.go

```diff
@@ -517,8 +517,7 @@ func (b *Builder) computeRoutes(sw *ObjectStatusWriter, proxy *projcontour.HTTPP
 				return nil
 			}
 
-			if !pathConditionsValid(include.Conditions) {
-				sw.SetInvalid("include: cannot specify multiple path conditions in the same include")
+			if !pathConditionsValid(include.Conditions, sw, "include") {
```

Comment:
suggestion: for consistency, please move sw to the front of the parameter list.

Reply:
Missed that, no problem.